packages/dart/npt_flutter/lib/features/profile_form/view/profile_form_view.dart

```diff
@@ -72,6 +72,24 @@ class ProfileFormView extends StatelessWidget {
                               ),
                             ),
                             gapH20,
+                            const Padding(
+                              padding: EdgeInsets.symmetric(horizontal: Sizes.p50),
+                              child: Column(
+                                crossAxisAlignment: CrossAxisAlignment.start,
+                                children: [
+                                  Text(
+                                    'Advanced Settings',
```

Comment:
TODO: localize this later, blocking next release to the stores, but I need this merged for another English only build first.